src/Transports.AspNetCore/Common/GraphQLRequestDeserializationResult.cs

```diff
@@ -0,0 +1,13 @@
+using GraphQL.Server.Common;
+
+namespace GraphQL.Server.Transports.AspNetCore.Common
+{
+    public class GraphQLRequestDeserializationResult
+    {
+        public bool WasSuccessful { get; set; }
```

Comment:
```suggestion
        public bool IsSuccessful { get; set; }
```

Reply:
Done. Also commented properties and type.